bin/windows/kotlin-server-petstore.bat

```diff
@@ -0,0 +1,10 @@
+set executable=.\modules\swagger-codegen-cli\target\swagger-codegen-cli.jar
+
+If Not Exist %executable% (
+  mvn clean package
+)
+
+REM set JAVA_OPTS=%JAVA_OPTS% -Xmx1024M -DloggerPath=conf/log4j.properties
+set ags=generate  --artifact-id "kotlin-petstore-server" -i modules\swagger-codegen\src\test\resources\2_0\petstore.yaml -l kotlin --library=ktor -o samples\server\petstore\kotlin
```

Comment:
Typo: `-l kotlin` => `-l kotlin-server`

Reply:
oops. thanks for catching that. I've made the change.